### PR TITLE
add elba manifest

### LINKED ISSUE #14: Adding Elba manifest
There's this new package manager for Idris: https://github.com/elba/elba/, might be useful to have TParsec indexed by it?

### PR DESCRIPTION
**Pull request by clayrat**
closes #14

### CODE REVIEW

File: elba.toml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+[package]
+name = "gallais/idris-tparsec"
```
Comment (gallais) [inline elba.toml]: Can we call it `tparsec`? This is the name I have used on [the wiki](https://github.com/idris-lang/Idris-dev/wiki/Libraries) and the ipkg file.

File: elba.toml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+version = "0.1.0"
+authors = ["G. Allais"]
```
Comment (gallais) [inline elba.toml]: I think you should be listed as a co-author.

File: elba.toml (hunk 1 above)
Comment (clayrat) [reply elba.toml]: Apparently not, the [manual says](https://elba.readthedocs.io/en/latest/usage/manifest.html):

> The name must contain a group (i.e. a namespace) and a name, separated by a slash, or else the manifest will fail to parse.

Comment (clayrat) [reply elba.toml]: We can use a different namespace than your handle, I guess, though

File: elba.toml (hunk 2 above)
Comment (clayrat) [reply elba.toml]: Ok! :)

**Comment by clayrat** [comment]
It's probably an overkill for now, it seems to build if I use the version on disk :)

File: elba.toml (hunk 1 above)
Comment (gallais) [reply elba.toml]: Eh. I guess if we have to abide by this format, the least confusing option is to stick to the repo's url.

Comment (clayrat) [reply elba.toml]: Ah, I can also shorten the name to `gallais/tparsec`, is that fine?

Comment (clayrat) [reply elba.toml]: Yeah maybe let's leave it same as the url though

**Pull request merged**